### ISSUE TITLE
Separate "Paradoxical respiration" and "Flail chest"

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11092]
**Preferred term label:**
Paradoxical respiration, currently https://hpo.jax.org/browse/term/HP:0030207
Flail chest needs a separate HPO code
**Synonyms**

**Definition (free text, please give PubMed ID)**
From our pulmonologist:

Flail chest is a physical exam finding that can be present in the setting of chest wall trauma or injury. When multiple adjacent ribs are fractured in 2 or more places, this can create a focal "free-floating" segment of the chest wall that moves independently from the rest of the chest. The only other entities on the differential for this are pathologic rib fractures (as opposed to trauma) and iatrogenic (post chest surgery), so the differential isn't particularly broad.

Paradoxical breathing is a deviation from the normal chest wall movements during inspiration and expiration. Typically, the diaphragm moves down on inspiration and the chest expands. On expiration, the diaphragm relaxes and moves upward and the chest is drawn inward. With paradoxical breathing, this pattern is reversed. While flail chest refers to a focal portion of the chest wall, paradoxical breathing refers to the overall breathing pattern and global chest movement. The differential includes obstruction of the airway, certain neurologic conditions affecting the muscles of breathing such as SMA, and fulminant respiratory failure from a variety of causes.

**Parent term (use hpo.jax.org/app)**
Paradoxical respiration is currently https://hpo.jax.org/browse/term/HP:0030207

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
As above.

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Thanks! 
I have created a new term Flail chest with the above definition (HP:0025003) and removed the false-synonym Flail chest from paradoxical breathing.

**Comment by MickeySegal** [comment]
UMLS has a flail chest finding, so adding that to the new HPO term could prevent a duplicate term in UMLS when they incorporate the new HPO term.

https://uts.nlm.nih.gov/uts/umls/concept/C0016196
